analyzers/pylint.py

```diff
@@ -72,10 +67,11 @@ def _handle_pylint_output(self, issues: str) -> int:
                 issue["message"] = issue["message"].replace("&lt;", "<").replace("&gt;", ">").replace("&amp;", "&")
                 issues_loads.append(issue)
 
+            utils.analyzers_output(self.settings.result_file, issues_loads)
             if issues_loads:
```

Comment:
In my opinion following code looks better: `return 1 if issue_loads else 0`

Reply:
In my opinion multi-line if looks more clear, I'd just remove extensive `else`